### ISSUE TITLE
Add numeric range

### ISSUE THREAD

**Issue by vasyugan** [huridocs/uwazi#1971]
I am right now adding an entity type "indigenous people", and there, the population numbers are often estimates with a min and a max, so it would be nice to have a property of numeric rage available (as for date range)

**Comment by txau** [comment]
A workaround for this issue may be adding to different numeric fields, one for the min and another for the max. Not as neat and straight forward as having a single field but may do the trick. Would that work for you?

**Comment by vasyugan** [comment]
I am not sure, depends on whether this is searchable. That is e.g. find me all indigenous peoples with an estimated population range between 100,000 and 150,000. Since what you suggest are two different properties, it would at least two different searches, wouldn't it?

**Comment by txau** [comment]
That is correct. At the same time the kind of search we are talking about is pretty straight forward. If you activate the "Use as filter" option for these two options (min, max) in that template settings, you'll get two filters in the side panel.

These two filters look sort of the same way a combined filter for a single numeric-range filter would look like, allowing for from/to values for both the min and the max. So you could run any of these searches:

- A minimum population of at least X
- A minimum population of up to Y
- A minimum population between X and Y
- A maximum population of at least X
- A maximum population of up to Y
- A maximum population between X and Y

And any combination of these options, such as:

- A minimum population of at least X AND a maximum population of up to Y.

![screenshot from 2018-10-10 13-08-54](https://user-images.githubusercontent.com/489831/46732405-ab3a9b80-cc8d-11e8-9669-1cc12f2dfde2.png)
